### ISSUE TITLE
[BUG] Translation issue on Markdown fields

### ISSUE THREAD

**Issue by axeloz** [thedevdojo/voyager#4217]
## Version information

 - Laravel: v5.8
 - Voyager: v1.2.6
 - PHP: 7.2
 - Database: MariaDB

## Description

When a given field has type "markdown" in BREAD, the translation buttons don't change the markdown editor content. Whenever I click "EN" or "FR" into the edition form, the WYSYWYG content remains unchanged.

However:
- when I go to the "view" page, it does work fine. 
- when I set this field as "textarea", it does work fine.

## Steps To Reproduce

1. Set a field as Markdown
2. Add a content, translate it and save
3. Go back to the edit form. Click on the translation buttons. 

## Expected behavior
When clicking on the "EN" button, the content should display the english translation

## Screenshots
<img width="695" alt="failed_edit_en" src="https://user-images.githubusercontent.com/1597611/59918210-522e2480-9424-11e9-926f-1a9270c3b558.png">

<img width="706" alt="failed_edit_fr" src="https://user-images.githubusercontent.com/1597611/59918277-86094a00-9424-11e9-84a0-73188d4bf35e.png">

**Comment by fletch3555** [comment]
Would you be able to submit a PR with a test showing this failing? It would help identify the issue and help build out our test suite.

**Comment by axeloz** [comment]
Sorry @fletch3555 I won't have time to setup a new instance of Voyager with a test case. But it's quite simple to reproduce on any existing instance of Voyager. 

Into my form, I am able to find the hidden field containing my translations: 
```
<input type="hidden" data-i18n="true" name="content_i18n" id="content_i18n" value="{&quot;fr&quot;:&quot;Je suis français\n&quot;,&quot;en&quot;:&quot;I'm english&quot;}">
```
I did not find the fix yet but I suppose this is related to `vendor/tcg/voyager/resources/assets/js/multilingual.js`. 

I can see line 199:
```
           if (inpUsr.hasClass('richTextBox')) {
                var $_mce = tinymce.get('richtext'+inpUsr.prop('name'));
                $_val = $_mce.getContent();
            }
```
There is a special case for richTextBox. 
Shoudn't it be a special case for SimpleMDE Markdown editor too ? 

Thanks

**Comment by axeloz** [comment]
Any update on this one?

**Comment by fareesh** [comment]
Ran into this issue myself - submitted a PR with the fix I'm using.

**Comment by emptynick** [comment]
Fixed in #4361 (merged into 1.x)
